translations/ui-finance/en.json

```diff
@@ -442,7 +442,7 @@
   "transaction.description": "Description",
   "transaction.fiscalYear": "Fiscal year",
   "transaction.from": "From",
-  "transaction.fundValidation": "Transfers (allocations) can not be made from a budget to itself",
+  "transaction.fundValidation": "Transfers and allocations can not be made from one budget to that same budget",
```

Comment:
@dltj, could you please update translations for this? thanks.

Reply:
In progress with https://github.com/folio-org/ui-finance/pull/167.